src/Extractor/HttpClient.php

```diff
@@ -245,7 +233,7 @@ public function fetch($url, $skipTypeVerification = false, $httpHeader = [])
         // (regex found here: https://stackoverflow.com/a/137831/569101)
         preg_match_all('/<!--\[if\s(?:[^<]+|<(?!!\[endif\]-->))*<!\[endif\]-->/mi', $body, $matchesConditional);
 
-        if (isset($matchesConditional[0]) && \count($matchesConditional[0]) > 1) {
+        if (isset($matchesConditional[0]) && (is_countable($matchesConditional[0]) ? \count($matchesConditional[0]) : 0) > 1) {
```

Comment:
Hmm, I would expect `$matchesConditional` to always be array of arrays so it should be countable: https://www.php.net/manual/en/function.preg-match-all.php Maybe it is Rector/PHPStan bug.

Reply:
I guess so